autoinstaller/install.sh

```diff
@@ -62,12 +305,22 @@ VAI_install_xbps_keys() {
 }
 
 VAI_install_base_system() {
+    base_pkgs="base-system grub grub-x86_64-efi grub-i386-efi";
+
+    if [ "$has_lvm" = "1" ]; then
+        base_pkgs="${base_pkgs} lvm2";
+    fi;
+
+    if [ "$has_luks" = "1" ]; then
+        base_pkgs="${base_pkgs} cryptsetup";
+    fi;
+
     # Install a base system
-    XBPS_ARCH="${XBPS_ARCH}" xbps-install -Sy -R "${xbpsrepository}" -r /mnt base-system grub
+    XBPS_ARCH="${XBPS_ARCH}" xbps-install -Sy -R "${xbpsrepository}" -r /mnt $base_pkgs;
```

Comment:
please change this to use the brace notation that is used everywhere else ( ${base_pkgs} ).